.github/build/requirements.txt

```diff
@@ -0,0 +1,11 @@
+openshift-client==1.0.18
+rich==12.5.1
+ray[default]==2.1.0
+kubernetes==26.1.0
+codeflare-torchx==0.5.0.dev5
```

Comment:
I think it's this. Old version of the custom torchx. It should be 0.6.0.dev0

Reply:
Oh good catch, thanks!